src/components/MovieCard.js

```diff
@@ -34,3 +36,8 @@ const mapStateToProps = state => ({
 });
 
 export default connect(mapStateToProps, null)(MovieCard);
+
+MovieCard.propTypes = {
+allMovies: PropTypes.array,
+
```

Comment:
For movie card, wouldn't our PropTypes need to test the following?
`id,
    title,
    poster_path,
    backdrop_path,
    release_date,
    overview,
    average_rating`
rather than the allMovies `array?`

Reply:
Thanks for pointing this out. I think this PropTypes approach would apply in the situation where we're pulling MovieCard props in the form of the allMovies object from the store (we currently have it setup so that we're doubly passing props from the store as well as in a destructured format from the parent component).

I've cut out the props coming from the store and reformatted the proptypes to match your recommendation.

Good looks!
-AD